package.json

```diff
@@ -42,6 +42,6 @@
     "jest": "^26.1.0",
     "lerna": "^3.22.1",
     "prettier": "^2.0.5",
-    "typescript": "^3.9.3"
+    "typescript": "^3.9.7"
```

Comment:
Here and below, I just bumped all the `typescript` versions to the same version. Shouldn’t affect anything.